### PR TITLE
Core/overview quickstart

### PR DESCRIPTION
**Pull request by clairesunstudio**
- Moved up and created a quick start section in "Get Started"
- Add back mayflower theme for docs 

Screen Shot 2020-12-11 at 1 20 04 PM

### CODE REVIEW

File: packages/core/stories/about/Usage.stories.mdx [hunk 1]
```diff
@@ -1,20 +1,40 @@
...
+
+To start building with Mayflower in HTML, start by importing the global styles and then incrementally add the styles and JS for each Mayflower component:
+
+- To import fonts and get started with basic Mayflower styles, link to the compiled `css/global.css` (or the minified version, `css/global.min.css`) from the @massds/mayflower-assets package or the assets CDN.
+
+- To include basic page layout styles from Mayflower, link to the compiled `css/layout.css` (or the minified version, `css/layout.min.css`) from the @massds/mayflower-assets package or the assets CDN.
+
+- To use a Mayflower branding component on your site, go to the documentation of the component, copy and paste the HTML code snippet, and link to the CDN version of CSS and JS files in the appropriate sections of your HTML.
+
+> You should always link to the Mayflower CDN at a specific version. That way, you never have to worry about your website getting impacted by any Mayflower breaking updates, and you can upgrade to a new Mayflower version whenever your team is ready. Alternatively, you can host the CSS and JS downloaded from mayflower assets package.
```
Comment (clairesunstudio) [inline packages/core/stories/about/Usage.stories.mdx]:
```suggestion
> For a production site, you should always link to the Mayflower CDN at a specific version. That way, you never have to worry about your website getting impacted by any Mayflower breaking updates, and you can upgrade to a new Mayflower version whenever your team is ready. Alternatively, you can host the CSS and JS downloaded from mayflower assets package. 
```

File: packages/core/stories/about/Usage.stories.mdx [hunk 2]
```diff
@@ -111,21 +131,6 @@ For examples:
 
 <br />
 
-## Build with Mayflower
-
-To start building with Mayflower in HTML, start by importing the global styles and then incrementally add the styles and JS for each Mayflower component:
-
-- To import fonts and get started with basic Mayflower styles, link to the compiled `css/global.css` (or the minified version, `css/global.min.css`) from the @massds/mayflower-assets package or the assets CDN.
```
Comment (tkoleary) [inline packages/core/stories/about/Usage.stories.mdx:118]: Shouldn't we provide the full CDN links here and on 120?

File: packages/core/stories/about/Usage.stories.mdx [hunk 3]
```diff
@@ -1,20 +1,40 @@
...
 
-## Mayflower assets
+
+## Quick Start
+To kickstart an HTML page with Mayflower Header, Footer, basic layout and styles:
+<Button theme="c-primary-alt" onClick={linkTo(generateTitle('Template'))}>Start From a Template</Button>
+
+To start building with Mayflower in HTML, start by importing the global styles and then incrementally add the styles and JS for each Mayflower component:
+
+- To import fonts and get started with basic Mayflower styles, link to the compiled `css/global.css` (or the minified version, `css/global.min.css`) from the @massds/mayflower-assets package or the assets CDN.
```
Comment (tkoleary) [inline packages/core/stories/about/Usage.stories.mdx:24]: Shouldn't we provide the full CDN links here and on 26?

File: packages/core/stories/about/Usage.stories.mdx [hunk 4]
```diff
@@ -1,20 +1,40 @@
...
+
+- To include basic page layout styles from Mayflower, link to the compiled `css/layout.css` (or the minified version, `css/layout.min.css`) from the @massds/mayflower-assets package or the assets CDN.
+
+- To use a Mayflower branding component on your site, go to the documentation of the component, copy and paste the HTML code snippet, and link to the CDN version of CSS and JS files in the appropriate sections of your HTML.
+
+> You should always link to the Mayflower CDN at a specific version. That way, you never have to worry about your website getting impacted by any Mayflower breaking updates, and you can upgrade to a new Mayflower version whenever your team is ready. Alternatively, you can host the CSS and JS downloaded from mayflower assets package. 
+
+> To view a specific version of this documentation, go to https://mayflower.digital.mass.gov/core/v/:version
```
Comment (tkoleary) [inline packages/core/stories/about/Usage.stories.mdx:34]: There is a storybook addon for versioning. Should we use that?
https://www.npmjs.com/package/storybook-addon-versions

Comment (clairesunstudio) [reply packages/core/stories/about/Usage.stories.mdx:34]: hmm this doesn't seem be well supported, it was published 3 years ago not sure if it works with the latest storybook

File: packages/core/stories/about/Usage.stories.mdx (hunk 3 above)
Comment (clairesunstudio) [reply packages/core/stories/about/Usage.stories.mdx:24]: sure

**Pull request merged**